web/portal/src/main/webapp/WEB-INF/conf/pwa/pwa-configuration.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd" xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+
+  <component>
+    <key>PWAConfigurationProperties</key>
+    <type>org.exoplatform.container.ExtendedPropertyConfigurator</type>
+    <init-params>
+      <properties-param>
+        <name>PWAConfigurationProperties</name>
+        <description>PWA configuration properties</description>
+        <property name="pwa.service.worker.enabled" value="${pwa.service.worker.enabled:true}" />
+        <property name="pwa.service.worker.path" value="${pwa.service.worker.path:war:/conf/pwa/service-worker.js}" />
+        <property name="pwa.manifest.enabled" value="${pwa.manifest.enabled:true}" />
+        <property name="pwa.manifest.path" value="${pwa.manifest.path:war:/conf/pwa/manifest.json}" />
```

Comment:
If Manifest will be disabled, we could remove those configurations too

Reply:
Done, thanks @ahamdi